pages/web/web-paas/languages-java/migration/guide.en-asia.md

```diff
@@ -141,7 +146,8 @@ export DB_HOST=`echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".databas
 | [Spring Data JPA](https://community.platform.sh/t/how-to-overwrite-spring-data-variable-to-access-platform-sh-services/518) | [Source](https://github.com/platformsh-examples/java-overwrite-configuration/tree/master/spring-jpa) |
 | [Payara JPA](https://community.platform.sh/t/how-to-overwrite-variables-to-payara-jpa-access-platform-sh-sql-services/519) | [Source](https://github.com/platformsh-examples/java-overwrite-configuration/blob/master/payara/README.md) |
 
-To reduce the number of lines in the application file and to make it cleaner, you have the option to move the variable environment to another file: the `.environment` [file as part of your application](../../development-variables#shell-variables).
+To reduce the number of lines in the application file and to make it cleaner,
+you have the option to move the variable environment to another file: a [`.environment` file](.../../development-variables#shell-variables).
```

Comment:
Hello @sreedhards, this updated link won't work. It has to be reverted. Thanks

Reply:
Have verified it, it is working.